cs/configuring.texy

```diff
@@ -345,3 +344,17 @@ Například takto může vypadat definice služby pojmenované `database`, což
 services:
 	database: PDO('mysql:host=127.0.0.1;dbname=test', root, password)
 \--
+
+
+Rozšíření
+=========
+
+Pomocí rozšíření můžeme rozdělit naši aplikaci do malých nezavislých častí s vlastní konfigurací. Nette obsahuje řadu předdefinovaných rozšíření, o kterých se dočtete více v kapitole [DI: Základní rozšíření pro kontejner |di-defaultextensions].
```

Comment:
Rad je tam vyjmenuju :-) 

> Určitě neumí aplikaci rozdělit do malých nezávislých částí :-) A Nette obsahuje jen několik, ne řadu rozšíření.

Vadi ti tam to nezavislych? :-) Pripadne, je ta veta cela blbe? Ja mel za to, ze rozsireni jsou k tomu, ze ti tu aplikaci logicky naporcuje na maly casti.